fix/DOC-4317:Document the search and delete action popup (#127)
exoplatform/exo-documentation: docs/Administration.rst

Manage Domains
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> In the Manage Domains page, you will find a table of all Domains in the platform, a search field to search domains and filter button for enabled Domains status. 

Reviewer: ```suggestion
In the Manage Domains page, a table containing all the Domains in the platform is displayed. To ease the domain's management, a search field is available to allow you searching domains and a filter button to allow you filtering domains by status. 
```

Once the domain is deleted, all related badges and rules will be disabled and have empty fields in the Domain column in the Manage Rules and Manage Badges tables.

Search Domain
----------

> You can search a domain just by typing some characters on the search field just next to the Add Domain button. 

Reviewer: ```suggestion
You can search a domain just by typing some characters on the search field near to the ``Add Domain`` button. 
```

> Fields that will be take into account in the search are:

Reviewer: ```suggestion
Fields taken into account during the search are:
```

- title
- description

> The table will display only domains containing characters typed. 

Reviewer: ```suggestion
Search results contain only the domains having in their title or description the typed characters.
```

> .. note:: If no results found, the *Empty Table* information will display.

Reviewer: ```suggestion
.. note:: If no results found, an *Empty Table* information is displayed.
```

.. _ManageRules:

Manage Rules
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> In the Manage Rules page, you will find a table of all rules in the platform, a search field to search rules and filter button for enabled Rules status. 

Reviewer: ```suggestion
In the Manage Rules page, a table containing all the rules in the platform is displayed. To ease the rules' management,  a search field is available to allow you searching rules and a filter button to allow you filtering rules by status.  
```

Every rule have: 

- Rule: A pulldown displaying all predefined rules.

- Description: A description of the rule to explain its goal.

> - Points: The number of points obtained by the rule.

Reviewer: ```suggestion
- Points: The number of points gained by the rule.
```

- Domain: The domain of the rule. It could be **social**, **Teamwork**, **Knowledge**...

- Enabled: The status of the rule. It could be **True** for enabled status or **false** for disabled.

> - Actions: A delete action for rules is displayed for every line of Rules table. 

Reviewer: ```suggestion
- Actions: A delete action for each Rule in the table.
```

----------
You can edit a rule just by clicking on the edit icon of the rule you want to update. 

> A popup containing existing informations of the selected rule to edit will display with 2 buttons: 

Reviewer: ```suggestion
A popup containing the existing information of the selected rule to edit is displayed with 2 buttons: 
```

> - Confirm: To save changes that you made in the rule

Reviewer: ```suggestion
- Confirm: To save the made changes on the rule
```

- Cancel: To cancel changes and conserve the already existing rule.

|image326|

Delete Rule
----------

> You can delete a rule just by clicking on the delete icon of the rule you want to delete. 

Reviewer: ```suggestion
You can delete a rule just by clicking on the delete icon. 
```

> A confirmation popup will be displayed and you can confirm the rule's delete or cancel it.

Reviewer: ```suggestion
A confirmation popup is displayed allowing you to confirm or cancel the changes done.
```

.. note:: Once the rule is deleted, it will be not used anymore on the points score.

Search Rule
----------

> You can search a rule just by typing some characters on the search field just next to the Add Rule button. 

Reviewer: ```suggestion
You can search for a rule just by typing some characters in the search field near the Add Rule button. 
```

> Fields that will be take into account in the search are:

Reviewer: ```suggestion
Fields taken into account in the search are:
```

- Rule 
- Description
- Points

> The table will display only rules containing characters typed. 

Reviewer: ```suggestion
Search results contain only rules having in their Rule, description or Points fields the typed characters. 
```

> .. note:: If no results found, the *Empty Table* information will display.

Reviewer: ```suggestion
.. note:: If no results found, an *Empty Table* information is displayed.
```

You can add a new Badge just by clicking on **Add Badge** button above the Badges table.

> You will have a search field to search Badges and filter button for enabled badges status. 

Reviewer: ```suggestion
A search field is available allowing you to search for Badges and a filter button allowing you to filter them by status. 
```

Edit Badge
-----------

> You can edit a badge just by clicking on the edit icon of the badge you want to update. 

Reviewer: ```suggestion
You can edit a badge just by clicking on the edit icon of the corresponding badge. 
```

> A popup containing existing informations of the selected badge to edit will display with 2 buttons: 

Reviewer: ```suggestion
A popup containing the existing information of the selected badge to edit is displayed with 2 buttons: 
```

> - Confirm: To save changes that you made in the Badge

Reviewer: ```suggestion
- Confirm: To save the made changes
```

> - Cancel: To cancel changes and conserve the already existing badge.

Reviewer: ```suggestion
- Cancel: To cancel the made changes and keep the existing badge.
```

Delete Badge
----------

> You can delete a badge just by clicking on the delete icon of the badge you want to delete. 

Reviewer: ```suggestion
You can delete a badge just by clicking on the delete icon of the target badge to delete. 
```

> A confirmation popup will be displayed and you can confirm the badge's delete or cancel it.

Reviewer: ```suggestion
A confirmation popup is displayed allowing you to confirm the badge's deletion or cancel it.
```

.. note:: Once the badge is deleted, it will be not used anymore on the points score.

Search Badge
----------

> You can search a badge just by typing some characters on the search field just next to the Add Badge button. 

Reviewer: ```suggestion
You can search for a badge just by typing some characters on the search field near the Add Badge button. 
```

> Fields that will be take into account in the search are:

Reviewer: ```suggestion
Fields taken into account in the search are:
```

- Badge
- Description
- Score
- Domain

> The table will display only badges containing characters typed. 

Reviewer: ```suggestion
The table will display only badges containing the typed characters in one of the above cited fields. 
```

> .. note:: If no results found, the *Empty Table* information will display.

Reviewer: ```suggestion
.. note:: If no results found, an *Empty Table* information is displayed.
```
